applications/StructuralMechanicsApplication/tests/mixed_u_E_test/mixed_u_E_element_test_materials.json

```diff
@@ -0,0 +1,18 @@
+{
+    "properties" : [{
+        "model_part_name" : "Structure.Parts_Solid_body",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "LinearElasticPlaneStrain2DLaw"
+            },
+            "Variables"        : {
+                "DENSITY"       : 7850.0,
+                "YOUNG_MODULUS" : 206900000000.0,
+                "POISSON_RATIO" : 0.29,
+				"STABILIZATION_FACTOR" : 0.1
```

Comment:
Is this the constant inside the stabilization tau?

Reply:
Ah, is the factor to mix the strains right?

Reply:
indeed

Reply:
If so, I wouldn't provide it through the properties. Instead, I'd put it in the `solver_settings` of the `ProjectParameters.json` or directly hardcode it in the element header as a private static constexpr.